web/src/app/styles/materialStyles.ts

```diff
@@ -1,4 +1,7 @@
-export const styles = (theme) => ({
+import { Theme } from '@mui/material/styles'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const styles = (theme: Theme): any => ({
```

Comment:
I think this can be `CSSProperties` imported from `react`

```
import { CSSProperties } from 'react'

export const styles = (theme: Theme): CSSProperties => ({
```